api/PannerNode.json

```
@@ -79,7 +86,7 @@
               "version_added": "42"
             },
             "safari": {
-              "version_added": false
+              "version_added": "14.1"
             },
             "safari_ios": {
               "version_added": false
```

```suggestion
              "version_added": "14.5"
```

Confirmed on iOS 14.5.